pipelines/src/pipelines/utils/trigger_pipeline.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+from distutils.util import strtobool
+from google.cloud import aiplatform
+
+
+def trigger_pipeline(
+    template_path: str,
+    display_name: str,
+) -> aiplatform.PipelineJob:
+    """Trigger a Vertex Pipeline run from a (local) compiled pipeline definition.
+
+    Args:
+        template_path (str): file path to the compiled YAML pipeline
+        display_name (str): Display name to use for the PipelineJob
+
+    Returns:
+        aiplatform.PipelineJob: the Vertex PipelineJob object
+    """
+    project_id = os.environ["VERTEX_PROJECT_ID"]
+    location = os.environ["VERTEX_LOCATION"]
+    pipeline_root = os.environ["VERTEX_PIPELINE_ROOT"]
+    service_account = os.environ["VERTEX_SA_EMAIL"]
+
+    enable_caching = os.environ.get("enable_pipeline_caching", None)
```

Comment:
use upper case as we do for other env vars?

Reply:
I don't mind - this is an env variable that we set when running the command (`make run enable_pipeline_caching=true`) rather than one picked up from `env.sh`

Reply:
let's use upper case to standardise. also, you can change the code to:
```suggestion
    enable_caching = os.environ.get("enable_pipeline_caching")
```